src/libs/SidebarUtils.ts

```diff
@@ -457,7 +443,108 @@ function getOptionData({
     return result;
 }
 
+function getWelcomeMessage(report: OnyxEntry<Report>): WelcomeMessage {
+    const welcomeMessage: WelcomeMessage = {showReportName: true};
+    if (ReportUtils.isChatRoom(report)) {
+        return getRoomWelcomeMessage(report);
+    }
+
+    if (ReportUtils.isPolicyExpenseChat(report)) {
+        welcomeMessage.phrase1 = Localize.translateLocal('reportActionsView.beginningOfChatHistoryPolicyExpenseChatPartOne');
+        welcomeMessage.phrase2 = Localize.translateLocal('reportActionsView.beginningOfChatHistoryPolicyExpenseChatPartTwo');
+        welcomeMessage.phrase3 = Localize.translateLocal('reportActionsView.beginningOfChatHistoryPolicyExpenseChatPartThree');
+        if (report?.description) {
+            welcomeMessage.concatMessage = parseHtmlToText(report.description);
+        }
```

Comment:
We still have room for improvement, the phraseX messages won't be used if we have a description. Let's do the following:
- If we have description, set `welcomeMessage.messageHtml`
- Otherwise set `welcomeMessage.phraseX`

In both cases set `welcomeMessage.messageText` (rename from `concatMessage`)


Also in `ReportWelcomeText` instead of using `report/policy.description` use `welcomeMessage.messageHtml`